chipsec/modules/tools/secureboot/te.py

```diff
@@ -70,7 +71,7 @@ def dumpstr(s):
             print ("")
     print ("")
 
-#logger.VERBOSE = False
+#logger().VERBOSE = False
```

Comment:
Why are we fixing a comment?  Lets get rid of the dead code.

Reply:
Removed